SkietbaanBE/SkietbaanBE/Controllers/FeaturesController.cs

```diff
@@ -32,5 +32,34 @@ public User Search(string username)
             }
             return null;
         }
+
+        //// PUT: api/User/Update
+        [HttpPut]
+        [ActionName("Update")]
+        public async Task<IActionResult> PutUserMember([FromBody] User user)
```

Comment:
You use this endpoint like it adds new users but this is a put.  Should be a post if it adds new users.

Reply:
Its not adding new users though. Its changes an existing one.

Reply:
why do you have both these endpoints then